frontend/src/admin/components/sidebar/restart/restartServerButton.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./restartserverbutton.scss";
+
+export default function RestartServerButton() {
+    return (
+        <button className={"c-restart-server-button"}>
```

Comment:
The curly brackets are redundant. You don't have any dynamic content in the className string.
```suggestion
        <button className="c-restart-server-button">
```

Reply:
✅